tests/std/tests/VSO_0157762_feature_test_macros/test.cpp

```diff
@@ -175,6 +175,20 @@ STATIC_ASSERT(__cpp_lib_atomic_wait == 201907L);
 #endif
 #endif
 
+#if _HAS_CXX20
+#ifndef __cpp_lib_barrier
+#error __cpp_lib_barrier  is not defined

```

Comment:
There are two spaces before "is" here, in an apparent copy-paste relic. Occurs repeatedly below.